src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -4876,6 +4889,313 @@ namespace HeatBalanceSurfaceManager {
     // Beginning of Reporting subroutines for the HB Module
     // *****************************************************************************
 
+    void ReportThermalResilience() {
+
+        double c1 = -42.379;
+        double c2 = 2.04901523;
+        double c3 = 10.14333127;
+        double c4 = -.22475541;
+        double c5 = -.00683783;
+        double c6 = -.05481717;
+        double c7 = .00122874;
+        double c8 = .00085282;
+        double c9 = -.00000199;
+
+        int HINoBins = 5; // Heat Index range - number of bins
+        int HumidexNoBins = 5; // Humidex range - number of bins
+        int SETNoBins = 4; // SET report column numbers
+        static std::vector<double> lowSETLongestHours(NumOfZones, 0.0);
+        static std::vector<double> highSETLongestHours(NumOfZones, 0.0);
+        static std::vector<int> lowSETLongestStart(NumOfZones, 0.0);
+        static std::vector<int> highSETLongestStart(NumOfZones, 0.0);
+        static bool hasPierceSET = true;
+        static bool oneTimeFlag = true;
```

Comment:
You **cannot** use static flags inside functions.  There's no way to reset this in a clear_state because it is hidden inside a function.  There are these two plus two inside other Report* functions you added here, and they have to be extracted out and become variables on the namespace, and then they need to be cleared in a clear_state call.  This should fix the problem.

Reply:
I see. That makes sense. Thanks!